bokehjs/src/coffee/core/has_props.coffee

```diff
@@ -317,6 +317,7 @@ export class HasProps extends Backbone.Model
           immediate = v._immediate_references()
           for obj in immediate
             HasProps._value_record_references(obj, result, true) # true=recurse
+    else if v?.buffer instanceof ArrayBuffer
```

Comment:
I don't think ``v`` should ever be undefined here so I think the first clause should be sufficient.